packages/@ember/routing/route.ts

```diff
@@ -110,7 +110,7 @@ interface Route<T = unknown> extends IRoute<T> {
     ```
 
     You can also redirect elsewhere by calling
-    `this.transitionTo('elsewhere')` from within `willTransition`.
+    `router.transitionTo('elsewhere')` from within `willTransition`.
```

Comment:
```suggestion
    `this.router.transitionTo('elsewhere')` from within `willTransition`.
```

I would keep `this` everywhere, as you always have to inject the `router` service and access it via `this.router`.

Reply:
In the same doc I noticed `store.findRecord` in the [model](https://api.emberjs.com/ember/4.9/classes/Route/methods/model?anchor=model) hook section so I tried to keep it consistent. I can add `this` of course.

Reply:
Oh, didn't notice that. Based on the diff, switching from `this.transitionTo` to `router.transitionTo` felt / looked a bit weird.